scheduler/kubernetes/encrypted-volumes.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: page
+title: "Encrypted volumes using PVC"
+keywords: portworx, container, kubernetes, storage, k8s, flexvol, pv, persistent disk, encryption, pvc
+sidebar: home_sidebar
+meta-description: "Looking to use a encrypted volume with Kubernetes? Follow this step-by-step tutorial on how to provision encrypted volumes with Kubernetes."
+---
+
+* TOC
+{:toc}
+
+This document describes how to provision an encrypted volume using Kubernetes and Portworx. For more information on encryption refer the [Encrypted Volumes page](/manage/encrypted-volumes.html).
+
+Before you start using PVC encryption, you need to setup a secrets provider to store your secret keys and configure Portworx to use it. Refer the Encryption section for more details on setting up various secret stores.
```

Comment:
`Refer the Encryption section for more details on setting up various secret stores.`

Which section? We need to control the user's flow.